src/components/SourceTabs.js

```diff
@@ -124,12 +146,25 @@ const SourceTabs = React.createClass({
       click: () => showSource(tab)
     };
 
+    const copySourceUrl = {
+      id: "node-menu-close-tabs-to-right",
+      label: "Copy Link Address",
+      accesskey: "X",
+      disabled: false,
+      click: () => {
+        showSource(tab);
+        this.copyToTheClipboard();
```

Comment:
hmm, little confused why we're trying to show the source...

couldn't we do `this.copyToTheClipboard(tab.url);`